view/templates/admin_blocklist.tpl

```diff
@@ -0,0 +1,33 @@
+<script>
+	function confirm_delete(uname){
+		return confirm( "{{$confirm_delete}}".format(uname));
+	}
+</script>
+<div id="adminpage">
+  <h1>{{$title}} - {{$page}}</h1>
+  <p>{{$intro}}</p>
+	<p>{{$public}}</p>
+
+	<h2>{{$addtitle}}</h2>
+	<form action="{{$baseurl}}/admin/blocklist" method="post">
+    <input type='hidden' name='form_security_token' value='{{$form_security_token}}'>
```

Comment:
Standards: Inconsistent indentation through this file.
Please use double-quotes for attributes.